site/documentation/content/api/credentials/index.md

```diff
@@ -50,6 +50,11 @@ The body of the request MUST consist of a single *Data* section containing a UTF
 | :--------------- | :-------: | :--------- | :---------- |
 | *type*           | *yes*     | *string*   | The type of credentials to look up. Potential values include (but are not limited to) `psk`, `x509-cert`, `hashed-password` etc. |
 | *auth-id*        | *yes*     | *string*   | The authentication identifier to look up credentials for. |
+| *client-certificate* | *no*  | *string*   | The client certificate the device authenticated with. If present, it MUST be the DER encoding of the (validated) X.509 client certificate as a Base64 encoded byte array. |
+
+The *client-certificate* property MAY be used by the service implementation for auto-provisioning of devices. 
+To do so, the device registry needs to create credentials (and registration data) for the device if they do not already exist. 
+It SHOULD then return the newly created credentials in the response payload.
```

Comment:
Actually, it MUST return the credentials as specified further down the page. However, IMHO we should simply omit this sentence here because, as I already mentioned, the response payload requirements are stated further down anyway ...